standard_reports/encounters/2.6.0/argument_schemas/arguments.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
As per, 2.6 folder was the old "latest", can ignore these ones, see changes in the diff under "latest"